README.md

```diff
@@ -168,6 +169,11 @@ kubectl apply -k https://raw.githubusercontent.com/openzwave/zwave2mqtt/master/k
 
 4. Open the browser <http://localhost:8091>
 
+### Reverse Proxy Setup
+
+If you need to setup ZWave To MQTT behind a reverse proxy that needs a _subpath_ to
+work, take a look at [this](docs/subpath.md).
```

Comment:
semantic links for accessibility are better, imagine a screen reader shouting **THIS** at you when someone is tabbing through :)
```suggestion
work, take a look at [the reverse proxy configuraiton docs](docs/subpath.md).
```